README.md

```diff
@@ -1,91 +1,150 @@
+Package `eris` provides a better way to handle, trace, and log errors in Go.
 
 `go get github.com/rotisserie/eris`
 
-## How is eris different?
+## Why you'll want to switch to eris
+
+Named after the Greek goddess of strife and discord, this package is designed to give you more control over error handling via error wrapping, stack tracing, and output formatting. `eris` was inspired by a simple question: what if you could fix a bug without wasting time replicating the issue or digging through the code?
 
-Named after the Greek goddess of strife and discord, this package is intended to give you more control over error handling via error wrapping, stack tracing, and output formatting. Basic error wrapping was added in Go 1.13, but it omitted user-friendly `Wrap` methods and built-in stack tracing. Other error packages provide some of the features found in `eris` but without flexible control over error output formatting. This package provides default string and JSON formatters with options to control things like separators and stack trace output. However, it also provides an option to write custom formatters via [`eris.Unpack`](https://godoc.org/github.com/rotisserie/eris#Unpack).
+`eris` is intended to help developers diagnose issues faster. The [example](example_logger_test.go) that generated the output below simulates a realistic error handling scenario and demonstrates how to wrap and log errors with minimal effort. This specific error occurred because a user tried to access a file that can't be located, and the output shows a clear path from the source to the top of the call stack.
+
+```golang
+{
+  "error":{
+    "root":{
+      "message":"error internal server",
+      "stack":[
+        "eris_test.GetRelPath:/Users/roti/go/src/github.com/rotisserie/eris/example_logger_test.go:58",
+        "eris_test.ProcessResource:/Users/roti/go/src/github.com/rotisserie/eris/example_logger_test.go:79",
+        "eris_test.ProcessResource:/Users/roti/go/src/github.com/rotisserie/eris/example_logger_test.go:82",
+        "eris_test.Example_logger:/Users/roti/go/src/github.com/rotisserie/eris/example_logger_test.go:140",
+      ]
+    },
+    "wrap":[
+      {
+        "message":"Rel: can't make ./some/malformed/absolute/path/data.json relative to /Users/roti/",
+        "stack":"eris_test.GetRelPath:/Users/roti/go/src/github.com/rotisserie/eris/example_logger_test.go:58"
+      },
+      {
+        "message":"failed to get relative path for resource 'res2'",
+        "stack":"eris_test.ProcessResource:/Users/roti/go/src/github.com/rotisserie/eris/example_logger_test.go:82"
+      }
+    ]
+  },
+  "level":"error",
+  "method":"ProcessResource",
+  "msg":"method completed with error",
+  "time":"2020-01-12T13:50:00-05:00"
+}
+```
 
```

Comment:
might be a good idea to add another paragraph here (or move the "using eris" paragraph and modify it a bit. something to summarize a few of the reasons why users should use eris